packages/client/hmi-client/src/components/responsive-matrix/cell-line-graph.vue

```diff
@@ -114,13 +114,12 @@ export default {
 				return acc;
 			}, {});
 
-			for (let row = startRow, i = 0; row <= endRow; row++, i++) {
-				this.dataRowList[row].forEach((cell) => {
-					if (cell.col <= endCol && cell.col >= startCol) {
-						this.parameters.forEach((param) => selectedCells[param].push(cell[param]));
-					}
-				});
-			}
+			this.iterateSelectionIndexes([startRow, -1, endRow, -1], (idx) => {
+				const cell = this.dataCellList[idx];
+				if (cell.col <= endCol && cell.col >= startCol) {
+					this.parameters.forEach((param) => selectedCells[param].push(cell[param]));
+				}
+			});
```

Comment:
Maybe it is a naming thing, but it probably reads better if you don't have anonymous callback functions. Instead of
```
this. iterateSelectionIndexes([....], () => {
})
```  
Perhaps

```
this.iterateSelectionIndexes([...], some_meaningful_name)
```

Reply:
can do, i can turn this into a named method